manifests/v1beta1/installs/katib-with-kubeflow-cert-manager/patches/mysql-pvc.yaml

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
Do we have any options to combine this patch and https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-with-kubeflow/patches/mysql-pvc.yaml in the one file ? cc @yanniszark

Reply:
I've implemented `katib-with-kubeflow-cert-manager` to use `katib-with-kubeflow` as a base, which solves needing to duplicate this file.